src/index.prod.html

```diff
@@ -8,10 +8,10 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <title>{{ page_title }}</title>
   <meta content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, shrink-to-fit=no" name="viewport">
-  {% with client_default|add:"/css/{VENDOR_CSS}" as vendor_css %}
+  {% with client_default|add:"/css/vendor.min.css" as vendor_css %}
     <link rel="stylesheet" href="{% static vendor_css %}">
   {% endwith %}
-  {% with client_default|add:"/css/{APP_CSS}" as app_css %}
+  {% with client_default|add:"/css/app.min.css" as app_css %}
```

Comment:
@wlorenzetti then let us know if removing these hashes in filenames (eg. [g3w-admin/g3w-admin/client/templates/client/index.html#L14](https://github.com/g3w-suite/g3w-admin/blob/8c98b2082a8ca7e04a8880f02b625b7288552d32/g3w-admin/client/templates/client/index.html#L14)) could be a problem for the server-side static file caching

Reply:
@Raruto no. We need to be sure about cache.